### DISCUSSION TITLE
Updated documentation

### DISCUSSION THREAD

**Discussion by wiverson** [supabase-community/supabase-csharp#102] [Show and tell]
Hi folks!

I had some time to go through and update the documentation. I added a lot of material as well as splitting it up into multiple files to make it easier to update (and for others to send in PRs).   :)

You can view the updated docs on the main page - https://github.com/supabase-community/supabase-csharp

I'd like to send a thank you to @Phantom-KNA, @kaushalkumar86, and @salsa2k for sending in notes and gists. I'm mostly working with Unity on iOS and Android, and it's really cool to see folks also using stuff like Xamarin/MAUI/Blazor.

I'm sure there is more to be done, but it's nice to get a chunk out there. My short list for the next thing is to create a Unity template project based on the Unity docs I added, a video for that template, and also (once I get through app store approval) post some links showing off what I've been working on.

Cheers,
-Will

**Comment by Phantom-KNA** [comment]
Hi bro, remember to add OAuth(Google) Native Android MAUI/Xamarin [doc](https://gist.github.com/Phantom-KNA/9096bf443beaab0dd45cca5e9eaf4ccc). Thx for hard work.

**Reply by wiverson** [reply]
Added it at the bottom of this page - https://github.com/supabase-community/supabase-csharp/blob/master/Documentation/ThirdPartyOAuth.md

Also added your info to the bottom of https://github.com/supabase-community/supabase-csharp/blob/master/Documentation/DesktopClients.md

Awesome stuff!  :)

**Comment by kaushalkumar86** [comment]
Hi how are you?
Hope u remember me.
I implemented Supabase.Csharp in my unity-android project.
Postgrest, RPC with auth, Auth client all are working fine and as expected.
I even implemented Facebook auth with pkce flow and deep app linking and
it's also working great.
I was wondering, is there any way to do an anonymous signin? I know
Supabase supports it, but is there any way to do the same with CSharp? Or
if you guys are working on it or if there is anything I can do to
contribute for the same, please let me know.
